test/common/testlib.py

```diff
@@ -660,31 +659,50 @@ class MachineCase(unittest.TestCase):
     # These will be instantiated during setUp, and replaced with machine objects
     provision = None
 
+    @classmethod
+    def get_global_machine(klass, machine_options={}):
+        if not klass.global_machine:
+            klass.global_machine = klass.new_machine(klass, cleanup=False, **machine_options)
```

Comment:
the dictionary gets unrolled into arguments, but since it's always empty i think i can just remove all the options and just have `new_machine(klass, cleanup=False)`